scripts/local_fuzz_bots.py

```diff
@@ -53,19 +55,28 @@ def main(argv: Sequence[str] | None = None) -> None:
         )
         hyperdrive_pool = LocalHyperdrive(chain, hyperdrive_config)
 
+        raise_error_on_fail = False
+        if parsed_args.pause_on_invariance_fail:
+            raise_error_on_fail = True
+
         # TODO submit multiple transactions per block
-        run_local_fuzz_bots(
-            hyperdrive_pool,
-            check_invariance=True,
-            raise_error_on_failed_invariance_checks=False,
-            raise_error_on_crash=False,
-            log_to_rollbar=log_to_rollbar,
-            run_async=False,
-            random_advance_time=True,
-            random_variable_rate=True,
-            num_iterations=3000,
-            lp_share_price_test=parsed_args.lp_share_price_test,
-        )
+        try:
+            run_local_fuzz_bots(
+                hyperdrive_pool,
+                check_invariance=True,
+                raise_error_on_failed_invariance_checks=raise_error_on_fail,
+                raise_error_on_crash=False,
+                log_to_rollbar=log_to_rollbar,
+                run_async=False,
+                random_advance_time=True,
+                random_variable_rate=True,
+                num_iterations=3000,
+                lp_share_price_test=parsed_args.lp_share_price_test,
+            )
+
+        except FuzzAssertionException as e:
+            print(f"Pausing pool on fuzz assertion exception {repr(e)}.")
```

Comment:
we use `logging.info` in `testnet_fuzz_bot_invariant_checks`; should we use it here too?